.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,17 @@
 {
-  "name": "Sample Development Environment for CCF",
-  "context": "..",
+  "name": "CCF Development Environment",
   "image": "ccfmsrc.azurecr.io/ccf/ci:oe-0.18.2-virtual",
   "runArgs": [],
-  "extensions": ["ms-vscode.cpptools", "ms-python.python"]
+  "extensions": [
+    "ms-vscode.cpptools",
+    "ms-python.python",
+    "esbenp.prettier-vscode"
```

Comment:
I am a little worried about this, and I hope I'm wrong - here is why: we've had problems in the past caused by vscode not running the same version of a tool that we use in ci-checks.sh (mostly with cmake format, not with prettier). We pick up new versions directly from npm, whereas the plugin might lag behind a bit. When this happens, vscode produces code that doesn't pass CI checks.

Anyway, I'm happy to try this and see how it goes, so @ross-p-smith please close this and merge when you've read it.

Reply:
Yes - I can see the problem you outline. Ultimately, your CI will always run ci-checks so this will be the 'right' version. Adding this is just going to help those developers who use the vscode "Format" feature. But yes I see your point! :-)

I also won't be able to merge as I am not a maintainer - so one of your team will have to.